OpenEdXMobile/src/main/java/org/edx/mobile/model/discussion/DiscussionUtils.java

```diff
@@ -1,4 +1,4 @@
-package org.edx.mobile.discussion;
+package org.edx.mobile.model.discussion;
```

Comment:
Utils class shouldn't be under Model package.

Reply:
reverted